apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx

```diff
@@ -163,7 +166,11 @@ const FinalStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep, on
 									</ul>
 									<p>
 										{ __( 'Questions?', 'full-site-editing' ) }{ ' ' }
-										<Button isLink href="https://wordpress.com/help/contact" target="_blank">
+										<Button
+											isLink
+											href={ localizeUrl( 'https://wordpress.com/help/contact' ) }
```

Comment:
After #48645 you can tell `localizeUrl` if we are logged-in or logged-out:
```suggestion
											href={ localizeUrl( 'https://wordpress.com/help/contact', null, window.currentUser ) }
```